src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -298,7 +299,8 @@ func (cfg *Config) applyClientDefaults() {
 
 	if cfg.AutoLaunch == nil {
 		cfg.AutoLaunch = new(bool)
-		*cfg.AutoLaunch = false
+		*cfg.AutoLaunch = true
+		launcher.CreateLaunchFile(*cfg.AutoLaunch)
```

Comment:
Actually I think the above is what we want after all, as the only other way AutoLaunch should ever get populated is from the user, in which case CreateLaunchFile is called.